pkg/status/service_do_transition.go

```diff
@@ -0,0 +1,84 @@
+package status
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/configuration"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/http/rest"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/service"
+	"net/http"
+)
+
+type ServiceDoTransition struct {
+	issueId    string
+	statusName string
+}
+
+func (s *ServiceDoTransition) InitJiraAPI(params configuration.JiraAPIResourceParameters) (rest.JiraAPI, error) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ServiceDoTransition.InitJiraAPI should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5024349)